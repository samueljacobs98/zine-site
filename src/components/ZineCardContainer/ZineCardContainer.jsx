import "./ZineCardContainer.scss";
import ZineCard from "../ZineCard/ZineCard";
import me from "../../assets/images/me.jpeg";
import Zine from "../../assets/images/Zine.png";

const ZineCardContainer = () => {
  return (
    <ZineCard
      title={"title"}
      image={me}
      profileName={"profileName"}
      profileImage={Zine}
    />
  );
};

export default ZineCardContainer;
