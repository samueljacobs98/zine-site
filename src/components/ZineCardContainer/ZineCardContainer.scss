@use "../../styles//variables.scss" as *;
.zineCardContainer {
  display: grid;
  gap: 2rem;
  justify-items: center;
  grid-template-columns: repeat(1, minmax(0, 1fr));
}
@media only screen and (min-width: $tablet-breakpoint) {
  .zineCardContainer {
    grid-template-columns: repeat(2, minmax(0, 1fr));
  }
}

@media only screen and (min-width: $desktop-breakpoint) {
  .zineCardContainer {
    grid-template-columns: repeat(3, minmax(0, 1fr));
  }
} ;
