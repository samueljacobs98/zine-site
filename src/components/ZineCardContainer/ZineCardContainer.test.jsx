import { customRender } from "../../utils/testUtils";
import ZineCardContainer from "../../components/ZineCardContainer/ZineCardContainer";

it("should match ZineCardContainer component snapshot", () => {
  const { container } = customRender(<ZineCardContainer />);

  expect(container).toMatchSnapshot();
});
