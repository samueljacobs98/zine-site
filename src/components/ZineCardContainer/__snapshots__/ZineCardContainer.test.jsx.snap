// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match ZineCardContainer component snapshot 1`] = `
<div>
  <div
    class="zineCardContainer"
  >
    <div
      class="zineCard"
    >
      <div
        class="zineCard__leftContent"
      >
        <h2
          class="zineCard__leftContent-title"
        >
          CEREAL
        </h2>
        <h3
          class="zineCard__leftContent-profileName"
        >
          JOHN SMITCH
        </h3>
        <img
          alt=""
          class="zineCard__leftContent-profileImage"
          src="profile.png"
        />
      </div>
      <div
        class="zineCard__rightContent"
      >
        <img
          alt=""
          class="zineCard__zineImage"
          src="Zine.png"
        />
      </div>
    </div>
    <div
      class="zineCard"
    >
      <div
        class="zineCard__leftContent"
      >
        <h2
          class="zineCard__leftContent-title"
        >
          CEREAL
        </h2>
        <h3
          class="zineCard__leftContent-profileName"
        >
          JOHN SMITCH
        </h3>
        <img
          alt=""
          class="zineCard__leftContent-profileImage"
          src="profile.png"
        />
      </div>
      <div
        class="zineCard__rightContent"
      >
        <img
          alt=""
          class="zineCard__zineImage"
          src="Zine.png"
        />
      </div>
    </div>
    <div
      class="zineCard"
    >
      <div
        class="zineCard__leftContent"
      >
        <h2
          class="zineCard__leftContent-title"
        >
          CEREAL
        </h2>
        <h3
          class="zineCard__leftContent-profileName"
        >
          JOHN SMITCH
        </h3>
        <img
          alt=""
          class="zineCard__leftContent-profileImage"
          src="profile.png"
        />
      </div>
      <div
        class="zineCard__rightContent"
      >
        <img
          alt=""
          class="zineCard__zineImage"
          src="Zine.png"
        />
      </div>
    </div>
    <div
      class="zineCard"
    >
      <div
        class="zineCard__leftContent"
      >
        <h2
          class="zineCard__leftContent-title"
        >
          CEREAL
        </h2>
        <h3
          class="zineCard__leftContent-profileName"
        >
          JOHN SMITCH
        </h3>
        <img
          alt=""
          class="zineCard__leftContent-profileImage"
          src="profile.png"
        />
      </div>
      <div
        class="zineCard__rightContent"
      >
        <img
          alt=""
          class="zineCard__zineImage"
          src="Zine.png"
        />
      </div>
    </div>
    <div
      class="zineCard"
    >
      <div
        class="zineCard__leftContent"
      >
        <h2
          class="zineCard__leftContent-title"
        >
          CEREAL
        </h2>
        <h3
          class="zineCard__leftContent-profileName"
        >
          JOHN SMITCH
        </h3>
        <img
          alt=""
          class="zineCard__leftContent-profileImage"
          src="profile.png"
        />
      </div>
      <div
        class="zineCard__rightContent"
      >
        <img
          alt=""
          class="zineCard__zineImage"
          src="Zine.png"
        />
      </div>
    </div>
    ;
  </div>
</div>
`;
